chainstory/linkapp/templates/linkapp/lobby.html

```diff
@@ -1 +1,39 @@
-Hello there!
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Lobby</title>
+</head>
+<body>
+    <h1>Let's chat!</h1>
+    <input id="chat-message-input" type="text" size="100">
+    <input id="chat-message-submit" type="button" value="Send">
+    <div id="chat-log"></div>
+
+    <script>
+        const chatSocket = new WebSocket(
```

Comment:
Wow way easier than I thought it would be

Reply:
I was not upset to find that the case